src/message-poller.cpp

```diff
@@ -187,6 +189,12 @@ void MessagePoller::processNotification(const SyncNotification& notification)
         case SYNC_ERROR_ID_FOLDER_PERM_DENIED:
             msg = tr("Update to file %1 is denied by folder permission setting.").arg(path);
             break;
+        case SYNC_ERROR_ID_PERM_NOT_SYNCABLE:
+            msg = tr("No permission to sync this folder %1.").arg(path);
```

Comment:
`sync this folder %1` =>  `sync folder %1`

Reply:
OK